2022 - LA - Sons of Horus.cat

```diff
@@ -6345,7 +6345,7 @@ A Warlord with this Trait gains a bonus of +1 to its Leadership Characteristic w
               <profiles>
                 <profile id="e5a5-fd33-12ba-1647" name="Cast in Gold" publicationId="53a4-0d21-2f8a-2c45" page="211 (Updated in download)" hidden="false" typeId="a0e6-a7b4-d55d-85b8" typeName="Warlord Trait">
                   <characteristics>
-                    <characteristic name="Text" typeId="c68e-2cda-b67b-baca">When a Warlord with this Trait is engaged in a Challenge with an enemy model that has a Weapon Skill that is equal to or greater than the Weapon Skill of the Warlord with this Trait, then the Warlord with this Trait gains +1 Strength and +1 Attacks for the duration of that Challenge (if both models in the Challenge have this Trait, then both models gain the bonus regardless of the Weapon Skill of either model). Furthermore, if the Warlord with this Trait causes an enemy model with a Weapon Skill equal to or greater than the Weapon Skill of the Warlord with this Trait to be removed as a casualty during a Challenge, the controlling player of the Warlord with this Trait gains 1 Victory Point in addition to any gained from any other rule or mission objective. In addition, an army whose Warlord has this Trait may make an additional Reaction during the opposing player’s Movement phase as long as the Warlord has not been removed as a casualty.</characteristic>
+                    <characteristic name="Text" typeId="c68e-2cda-b67b-baca">When a Warlord with this Trait is engaged in a Challenge with an enemy model that has a Weapon Skill that is equal to or greater than the Weapon Skill of the Warlord with this Trait, then the Warlord with this Trait gains +1 Strength and +1 Attacks for the duration of that Challenge (if both models in the Challenge have this Trait, then both models gain the bonus regardless of the Weapon Skill of either model). Furthermore, if the Warlord with this Trait causes an enemy model with a Weapon Skill equal to or greater than the Weapon Skill of the Warlord with this Trait to be removed as a casualty during a Challenge, the controlling player of the Warlord with this Trait gains 1 Victory Point in addition to any gained from any other rule or mission objective. In addition, an army whose Warlord has this Trait may make an additional Reaction during the opposing player’s Movement phase as long as the Warlord has not been removed as a casualty.</characteristic>
```

Comment:
There is something off with this tag but I don't know what it is. Doesn't seem to be recognised as a proper tag.

Reply:
But that's just githubs's formatting, not anything to do with this PR or the data before/after, right?